public/js/pickup_delivery.js

```diff
@@ -484,13 +484,13 @@ $(document).ready(function() {
           var lastStop = stops[stops.length - 1];
           if (lastStop && lastStop[0] == ref && activity.pickup_shipment_id) {
             if (d)
-              lastStop[10] = lastStop[10] ? (duration(lastStop[10]) + d).toHHMMSS() : d.toHHMMSS();
+              lastStop[10] = lastStop[10] ? (lastStop[10] + d).toHHMMSS() : d.toHHMMSS();
```

Comment:
The fields with a call to `toHHMMSS()` method should maintain their call to `duration`